website/content/docs/networking/index.mdx

```diff
@@ -0,0 +1,330 @@
+Nomad, some of the different patterns and configurations you are likely to find
+and use, and how Nomad differs from other tools in this aspect.
+
+## Allocation networking
+
+The base unit of scheduling in Nomad is an [allocation], which means that all
+tasks in the same allocation run in the same client and share common resources,
+such as disk and networking. Allocations can request access to network
+resources, such as ports, using the [`network`][jobspec_network] block. At its
+simplest configuration, a `network` block can be defined as:
+
+```hcl
+job "..." {
+  # ...
+  group "..." {
+    network {
+      port "http" {}
+    }
+    # ...
+  }
+}
+```
+
+Nomad reserves a random port in the client between [`min_dynamic_port`] and
+[`max_dynamic_port`] that has not been allocated yet and creates a port mapping
+from the host network interface to the allocation.
+
+{/* Source: https://drive.google.com/file/d/1q4a2ab0TyLEPdWiO2DIianAPWuPqLqZ4/view?usp=share_link */}
+[![Nomad Port Mapping](/img/networking/port_mapping.png)](/img/networking/port_mapping.png)
+
+The selected port number can be accessed by tasks using the
+[`NOMAD_PORT_<label>`][runtime_environment] environment variable to bind and
+expose the workload at the client's IP address and the given port.
+
+The specific configuration process depends on what you are running, but it is
+usually done using a configuration file rendered from a [`template`] or passed
+directly via command line arguments:
+
+```hcl
+job "..." {
+  # ...
+  group "..." {
+    network {
+      port "http" {}
+    }
+
+    task "..." {
+      # ...
+      config {
+        args = [
+          "--port=${NOMAD_PORT_http}",
+        ]
+      }
+    }
+  }
+}
+```
+
+It is also possible to request a specific port number, instead of a random one,
+by setting a [`static`] value for the `port`. **This should only be used by
+specialized workloads**, such as load balancers and system jobs, since it can
+be hard to manage them manually to avoid scheduling collisions.
+
+With the task listening at one of the client's ports, other processes can
+access it directly using the client's IP and port, but first they need to find
+these values. This process is called [service discovery].
+
+When using IP and port to connect allocations it is important to make sure your
+network topology and routing configuration allow the Nomad clients to
+communicate with each other.
+
+## Bridge networking
+
+Linux clients support a network [`mode`][network_mode] called [`bridge`]. A
+bridge network acts like a virtual network switch allowing processes connected
+to the bridge to reach each other while isolating them from others.
+
+When a Nomad agent starts in a Linux system it creates a bridge network called
+`nomad` (or the value set in [`bridge_network_name`]) using the [`bridge` CNI
+plugin][cni_bridge]. Before using this mode you must first [install the CNI
+plugins][cni_install] into your clients.
```

Comment:
This should like to https://github.com/hashicorp/tutorials/pull/1005 (internal repo) once the new content is available.